notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        df["test_class"] = df["class_probability"].apply(lambda x: 1 if x&gt;=0.5 else 0)
</pre><p>You are counting 0.5 as a crop here is that the intended behavior?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Yes, this was an assumption that I was making. Not sure if this is the best approach for our purposes - should we threshold another way (i.e [0, 0.5] for non-crop, (0.5, 1] for crop)?</p><p> </p><p>In any case, I will make a note about how thresholding is done in the notebook. </p>